test/library/packages/Itertools/cycle/check_non_indexable.chpl

```diff
@@ -0,0 +1,11 @@
+use Itertools;
+
+var A: [1..10] int;
+forall (id, el) in zip(1..10, cycle(1..4)) do A[id] = el;
+writeln(A);
+forall (id, el) in zip(1..10, cycle({1..6})) do A[id] = el;
+writeln(A);
+
+var B: [2..11] real;
+forall (id, el) in zip(2..11, cycle(1..6)) do B[id] = el:real;
```

Comment:
I'm not sure this array being of type `real` impacts this particular test of `cycle()`. (Sorry if I suggested this test)

Reply:
Yes, it doesn't impact the test. I will remove it in the next commit. :+1: